pom.xml

```diff
@@ -21,6 +21,21 @@ limitations under the License.
     <artifactId>bigtable-client</artifactId>
     <version>0.1.7-SNAPSHOT</version>
     <packaging>pom</packaging>
+    <name>${project.groupId}:${project.artifactId}</name>
+    <url>https://cloud.google.com/bigtable/</url>
+    <description>
```

Comment:
This seems like marketing copy instead of "A client to access Google Cloud Bigtable (http://...) from HBase".

It might also make sense to have descriptions in the submodules if the intent is for them to be referenced separately (e.g., the lightweight client in bigtable-grpc).